docs/content/latest/api/ycql/expr_fcall.md

```diff
@@ -89,7 +89,35 @@ CAST function converts the value returned from a table column to the specified d
 | `TIMEUUID` | `DATE`, `TIMESTAMP` |
 
 
-## Partition_hash function
+## Semantics
```

Comment:
what are these changes doing in this PR?

Reply:
had duplicate sections when merging, fixed